js/app.js

```diff
@@ -0,0 +1,234 @@
+console.log("works");
+//******************ANSWER THE FOLLOWING******************
+//1 a light switch with on/off
+//  I would store it in an object function by using lightSwitch as a key
+// and then function(){// turn on //turn off}
```

Comment:
was just looking for boolean -- you don't actually say boolean here... i like your idea though